<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="6" column="0" colspan="3">
     <widget class="QGroupBox" name="dataSend">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>数据发送</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,0">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QTextEdit" name="message">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>71</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="send">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>71</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>71</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QCheckBox" name="openCom">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>81</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>81</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>打开串口</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="5" colspan="2">
     <widget class="QGroupBox" name="dataLog">
      <property name="minimumSize">
       <size>
        <width>641</width>
        <height>451</height>
       </size>
      </property>
      <property name="title">
       <string>数据日志</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QTextBrowser" name="logBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>159</width>
        <height>83</height>
       </size>
      </property>
      <property name="title">
       <string>接收设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>4</number>
           </property>
           <item>
            <widget class="QRadioButton" name="rx_ascii">
             <property name="text">
              <string>ASCII</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rx_hex">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>HEX</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="saveFile">
           <property name="text">
            <string>接收保存到文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>159</width>
        <height>83</height>
       </size>
      </property>
      <property name="title">
       <string>发送设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QRadioButton" name="send_ascii">
             <property name="text">
              <string>ASCII</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="send_hex">
             <property name="text">
              <string>HEX</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="send_circle">
           <property name="text">
            <string>循环发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="serialConfig">
      <property name="minimumSize">
       <size>
        <width>159</width>
        <height>176</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>159</width>
        <height>176</height>
       </size>
      </property>
      <property name="title">
       <string>串口设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>串口号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="portName">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>20</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>波特率</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="baudRate">
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>115200</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>9600</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>数据位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="dataBits">
             <item>
              <property name="text">
               <string>8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>校验位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="parity">
             <item>
              <property name="text">
               <string>NONE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ODD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EVEN</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>停止位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="stopBits">
             <property name="editable">
              <bool>false</bool>
             </property>
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>249</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="clearRxLog">
        <property name="minimumSize">
         <size>
          <width>78</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>78</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>清除接收</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearTxLog">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>清除发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="maximumSize">
       <size>
        <width>159</width>
        <height>81</height>
       </size>
      </property>
      <property name="title">
       <string>数据可视化</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="signal_quality_button">
           <property name="text">
            <string>接收信号强度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="send_circle_2">
           <property name="text">
            <string>待定</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>217</width>
        <height>26</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QRadioButton" name="elevator_mode">
         <property name="minimumSize">
          <size>
           <width>91</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>91</width>
           <height>19</height>
          </size>
         </property>
         <property name="text">
          <string>乘梯模式</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="signal_quality">
         <property name="minimumSize">
          <size>
           <width>117</width>
           <height>19</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>117</width>
           <height>19</height>
          </size>
         </property>
         <property name="text">
          <string>信号质量检测</string>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
